packages/react/src/auth/SignInForm.tsx

```diff
@@ -129,7 +129,18 @@ export function SignInForm(props: SignInFormProps): JSX.Element {
             </AuthenticationForm>
           );
         } else if (mfaRequired) {
-          return <MfaForm login={login} handleAuthResponse={handleAuthResponse} />;
+          return (
+            <MfaForm
```

Comment:
I did change the API for `MfaForm`, but I do think this one is more generic and could be more useful to consumers other than `SignInForm`. Perhaps we can wait until v4 if we think this is an unacceptable breaking change since this component is actually exported?